modules/ROOT/pages/install-workflow.adoc

```diff
@@ -3,14 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-After your infrastructure team verifies that the required infrastructure is correctly installed and configured as described in xref:install-checklist.adoc[Product Prerequisites], you can install Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) using one of the following installers: 
+Have your infrastructure team verify that the required infrastructure is correctly installed and configured as described in xref:install-checklist.adoc[Product Prerequisites]. 
 
-* xref:install-installer.adoc[Run the GUI installer]
-* xref:install-auto-install.adoc[Run the CLI installer]
-
-[WARNING]
-You must contact MuleSoft Professional Services prior to installation. MuleSoft professional services must be involved with your installation procedure.
-
-The GUI installer is interactive and walks you through a set of wizard pages that enable you to specify configuration options, but it also requires you to open extra ports and a web browser to access the pages.
-
-The CLI installer does not require human interaction using a web browser and is useful for automation.
+[IMPORTANT]
```

Comment:
Should this be it's own section with a header? Something like:

### MuleSoft Professional Services Required
MuleSoft professional services must be involved throughout the installation process. Prior to installation, have your infrastructure team verify that the required infrastructure is correctly installed and configured as described in xref:install-checklist.adoc[Product Prerequisites].

Reply:
Got it - thanks!